### ISSUE TITLE
How do we measure accuracy? What to implement? 

### ISSUE THREAD

**Issue by MUCDK** [theislab/moscot#59]
Possible methods for sequential data:
- similar to WOT: interpolate P_t by P_{t-1} and p_{t+1}. Here, we could also take the gradient flows provided by OTT
- similar to moscot-lineage: compare calculated transport matrix row- or column-wise to ground truth transport matrix (only works if we know ground truth, i.e. with simulated data), do I understand this correctly?
- new approach: leave cells out for training and project cells to subspace spanned by training data points. Apply optimal transport to linear combination
